config/locales/diaspora/en.yml

```diff
@@ -32,6 +32,7 @@ en:
   no_results: "No results found"
   _contacts: "Contacts"
   _terms: "Terms"
+  _podmin_mail: "Podmin Mail"
```

Comment:
I think "Contact Podmin" would be a better text for most users, because "Contact" is what most people understand (also by people not very familiar with the project) and that's also what I'm looking for when I search how to contact somebody.